src/randoop/reflection/ReflectionPredicate.java

```diff
@@ -11,10 +11,36 @@
  */
 public interface ReflectionPredicate {
 
+  /**
+   * Indicate whether a class meets the use criteria for predicate.
+   * 
+   * @param c  the class to test.
+   * @return true if the class meets the predicate criteria, false, otherwise.
+   */
   public boolean canUse(Class<?> c);
```

Comment:
The code before used the name "filter" and said when we "can use" a reflective object.
I changed the terminology to predicates, and kept the method names.  I am open to changing them, but the calling methods are from Operation and correspond to saying
  "this operation satisfies the predicate"
while these methods are saying
  "this predicate is satisfied by the operation".

The Java 8 Predicate has the method test(T), which would be
  "this predicate tests true/false on the operation"

I'll go with the Java 8 convention.